03_myfavorite/src/Components/ResultPage.jsx

```diff
@@ -0,0 +1,77 @@
+import styled from 'styled-components';
+import Card from './Card';
+import crown from '../img/crown.png';
+
+function ResultPage({ winner }) {
+  return (
+    <Wrapper>
+      <WinnerIs>대망의 우승자는...!</WinnerIs>
+      <CardWrapper>
+        <Card name={winner.name} src={winner.src} />
+        <Crown src={crown} />
+        <ButtonContainer>
+          <RestartButton>다시하기</RestartButton>
+          <ShareButton>공유하기</ShareButton>
```

Comment:
`button` 태그에는 웬만해서 `type="button"` 을 명시해주자!
기본 타입이 `submit` 이기 때문에 input 태그 내에서 쓸 경우에 페이지 새로고침이 일어나게 돼
그래도 시멘틱하게 정확한 타입을 명시해주는 게 좋겠지??

[관련 아티클](https://nykim.work/96)

```suggestion
          <RestartButton type="button">다시하기</RestartButton>
          <ShareButton type="button">공유하기</ShareButton>
```